source/content/guides/wordpress-git/01-introduction.md

```diff
@@ -17,6 +17,8 @@ editpath: wordpress-Git/01-introduction.md
 reviewed: "2021-02-26"
 ---
 
+<Wistia src="7gn1g3ljz6" />
```

Comment:
@stevector the placement here feels off, mostly I think it's the title card / thumbnail preview "Content Changes" isn't right for a video on how to edit and deploy code. The video is effectively a combo of `/guides/wordpress-git/plugins/` with a sprinkle of `/guides/wordpress-git/next-steps/` (deploy) which is why I thought maybe it might work in the intro page instead maybe but it feels out of place. Feedback here especially welcomed

Reply:
Yes, and the spoken words at the beginning, "welcome to the 3rd and final..." feel out of place. I'd be fine with skipping this video.